### ISSUE TITLE
Doubles Tournaments

### ISSUE THREAD

**Issue by enforcer574** [enforcer574/smashclub#85]
Technical details: This must be implemented AFTER automatic bracket generation.
- For doubles, instead of using UUIDs in the tourney_match_order table, use group numbers.
- Add a table: tourney_player_groups, which links users to groups.

**Comment by SSMGrimm** [comment]
Need to have autogen done before starting.
Possibly a dropdown on Events page to pick teammate during signup.

**Comment by enforcer574** [comment]
This has been postponed to SmashClub 2.6 and will be part of its own update. Several parts of the app will need rework, including the database, the bracket, signup forms, and rank calculations.
